sdkproject/Assets/Mapbox/Unity/Editor/PropertyDrawers/LineGeometryOptionsDrawer.cs

```diff
@@ -16,8 +16,8 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
 			EditorGUI.BeginChangeCheck();
 
 			EditorGUILayout.PropertyField(property.FindPropertyRelative("Width"));
-			property.FindPropertyRelative("JoinType").enumValueIndex = (int)((LineJoinType) EditorGUILayout.EnumPopup("Join Type", (LineJoinType)property.FindPropertyRelative("JoinType").intValue));
-			if (property.FindPropertyRelative("JoinType").enumValueIndex == (int)LineJoinType.Miter)
+			property.FindPropertyRelative("JoinType").enumValueIndex = (int)((LineJoinType)EditorGUILayout.EnumPopup("Join Type", (LineJoinType)property.FindPropertyRelative("JoinType").intValue));
+			if (property.FindPropertyRelative("JoinType").enumValueIndex == (int)LineJoinType.Miter || property.FindPropertyRelative("JoinType").enumValueIndex == (int)LineJoinType.Bevel)
```

Comment:
@atripathi-mb I did it just for Miter as Studio also disables it for Bevel join type. Have you find any issues? Because if so there might be another problem somewhere else.

Reply:
@brnkhy miterLimit is being used to switch Bevel to Miter. If a user changes miter limit and then changes the join type to Bevel, there is no way to control the limit.

Reply:
@atripathi-mb I think it originally controls switching miter to bevel but yes there's also an if check (probably just in case) for vice versa as well. Again, checking mapbox studio, it hides miter limit on bevel and only enables it on miter. as you said it's still used for bevel but maybe they didn't want it to be customizable in bevel for some reason?

Reply:
@brnkhy Hiding a parameter that affects the rendering of a feature seems bad design to me. Could you verify with Studio team about the reasoning behind hiding the parameter ?

Reply:
Merging this PR . If we need to change the miter limit in the UI, let's do a separate PR.